### DISCUSSION TITLE
The plugin is not working in Ubuntu

### DISCUSSION THREAD

**Discussion by microts159** [WarmUpTill/SceneSwitcher#683] [General]
I installed the plugin in the ~/.config/obs-studio/plugins/advanced-scene-switcher/bin/64bit directory and it still doesn't load in OBS. I'll leave a log here to explain better: [log.txt](https://github.com/WarmUpTill/SceneSwitcher/files/10777618/log.txt)

I apologize if I have mistakes in my English, I'm using the translator.

**Comment by WarmUpTill** [comment]
The plugin is installed in the correct location but it cannot be loaded due to a missing dependency:
```
17:52:52.944: os_dlopen(/home/ubuntu/.config/obs-studio/plugins/advanced-scene-switcher/bin/64bit/advanced-scene-switcher.so->/home/ubuntu/.config/obs-studio/plugins/advanced-scene-switcher/bin/64bit/advanced-scene-switcher.so): /home/ubuntu/.config/obs-studio/plugins/advanced-scene-switcher/bin/64bit/advanced-scene-switcher.so: cannot open shared object file: No such file or directory
17:52:52.944: 
17:52:52.944: Module '/home/ubuntu/.config/obs-studio/plugins/advanced-scene-switcher/bin/64bit/advanced-scene-switcher.so' not loaded
```

I think the issue is that you are using a version of OBS that is built using Qt5 while the plugin is built for Qt6.
Unfortunately versions of this plugin using Qt5 are no longer supplied since OBS switched to Qt6 for most platforms with OBS 28.

You have two options:
* Switch to an older version of the plugin still built for OBS 27 / Qt5.
* Build the current version of the plugin yourself for Qt5.

For the former you will have to stick to [this version](https://github.com/WarmUpTill/SceneSwitcher/releases/tag/1.17.7).
Instructions to build the plugin yourself can be found [here](https://github.com/WarmUpTill/SceneSwitcher/blob/master/BUILDING.md).

Hope this helps!
Let me know if you have any questions!
